src/aws-cpp-sdk-core/include/aws/core/Region.h

```diff
@@ -16,44 +15,48 @@ namespace Aws
      */
     namespace Region
     {
-        // AWS_GLOBAL is a pseudo region that can be used to tell SDK to use the service global endpoint if there is any.
```

Comment:
do you want to keep that comment about global?

Reply:
nah now that theres several global regions, and putting the global region as the first entry of the enum adds a bit of weird custom logic that makes this change a bit messy.

also [existing AWS documentation on this exists that users could refer to](https://docs.aws.amazon.com/sdkref/latest/guide/feature-region.html).

> aws-global specifies the global endpoint for services that support a separate global endpoint in addition to Regional endpoints, such as AWS Security Token Service (AWS STS) and Amazon Simple Storage Service (Amazon S3).